cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverter.java

```diff
@@ -29,13 +30,10 @@
  * Common utility methods for dealing with HBase table name conversions.
  */
 public abstract class HTableNameConverter {
-  private String getHBaseTableName(String tableName) {
-    return encodeTableName(tableName);
-  }
 
-  private String encodeTableName(String tableName) {
+  public String encodeHBaseEntity(String entityName) {
```

Comment:
now that this is public, add a javadoc to explain what entities it encodes (table and namespace)?